assets/global.js

```diff
@@ -878,9 +886,13 @@ class VariantSelects extends HTMLElement {
     const addButton = button.querySelector('[name="add"]');
     const addButtonText = button.querySelector('[name="add"] > span');
     const price = document.getElementById(`price-${this.dataset.section}`);
+    const inventory = document.getElementById(`Inventory-${this.dataset.section}`);
+
     if (!addButton) return;
     addButtonText.textContent = window.variantStrings.unavailable;
     if (price) price.classList.add('visibility-hidden');
+    if (inventory) inventory.classList.add('visibility-hidden'), inventory.setAttribute('aria-hidden', 'true');
```

Comment:
I don't believe the `aria-hidden` part here is actually needed. Because it's running for unavailable variants, it never re renders anything. And that means no change to what's bing announced. So in this scenario we don't need it.